### PR TITLE
redesign yes/no toggle to be a11y compliant.

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

**Review by dartajax — CHANGES_REQUESTED** [review]
not a big deal but I noticed the blue outlining of the toggles does not disappear on lost focus ...

image

**Comment by stopfstedt** [comment]
> not a big deal but I noticed the blue outlining of the toggles does not disappear on lost focus ...
> 
does not appear in FF, but confirmed in Chrome. i added a rule to the stylesheet that will suppress this outline. :heavy_check_mark:

**Review by dartajax — APPROVED** [review]

File: tests/integration/components/toggle-yesno-test.js [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-    assert.expect(2);
-    const state = 'input';
-
     this.set('value', true);
-    await render(hbs`<ToggleYesno @yes={{value}} @action="clicked" />`);
-
-    assert.dom(state).isChecked();
-
+    await render(hbs`<ToggleYesno @yes={{this.value}} @action={{noop}} />`);
+    await a11yAudit(this.element);
```
Comment (jrjohnson) [inline tests/integration/components/toggle-yesno-test.js:16]: 💯

**Review by jrjohnson — APPROVED** [review]
!! 😤

**Pull request merged**